src/ngx_http_lua_ssl_certby.c

```diff
@@ -982,6 +982,51 @@ ngx_http_lua_ffi_priv_key_pem_to_der(const u_char *pem, size_t pem_len,
 }
 
 
+int
+ngx_http_lua_ffi_priv_key_pem_to_der_with_password(const u_char *pem,
+    size_t pem_len, const u_char *pwd, size_t pwd_len, u_char *der, char **err)
```

Comment:
@doujiang24 I have considered the `ngx_http_lua_ffi_priv_key_pem`. But there are some existing tests. I'm not sure it's worth to change the `ngx_http_lua_ffi_priv_key_pem`. So  just add the new api `ngx_http_lua_ffi_priv_key_pem_to_der_with_password`, then reimplement the Lua FFI API `ngx.ssl. priv_key_pem_to_der`

So if we almost agree to reimplement the `ngx_http_lua_ffi_priv_key_pem`, I will try to do :D